.gitignore

```diff
@@ -7,3 +7,4 @@ package.xml
 composer.lock
 experiments/
 vendor/
+.idea/
```

Comment:
Can you remove this added line from `.gitignore` and restore the missing line feed?